i18n/ads-language-pack-de/package.json

```diff
@@ -2,16 +2,15 @@
 	"name": "ads-language-pack-de",
 	"displayName": "German Language Pack for Azure Data Studio",
 	"description": "Language pack extension for German",
-	"version": "1.26.0",
+	"version": "1.29.0",
 	"publisher": "Microsoft",
 	"repository": {
 		"type": "git",
-		"url": "https://github.com/Microsoft/azuredatastudio"
+		"url": "https://github.com/Microsoft/vscode-loc"
 	},
 	"license": "SEE SOURCE EULA LICENSE IN LICENSE.txt",
 	"engines": {
-		"vscode": "^1.34.0",
-		"azdata": ">=1.26.0"
+		"vscode": "^1.52.0"
```

Comment:
These should specify the azdata engine version (`"azdata": ">=1.29.0"`) and should really just leave the vscode version as * since it doesn't really matter for our purposes what the VS Code engine version is.